example/dns.go

```diff
@@ -0,0 +1,59 @@
+package example
```

Comment:
Not for this PR, but we don't have cc headers. Do we want to add them to this project? There has been talks about removing them everywhere all together.